radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/substatehash/SubstateAccumulatorHashModule.java

```diff
@@ -101,5 +105,6 @@ protected void configure() {
     bindConstant()
         .annotatedWith(Names.named(VERIFY_EPOCH_HASH_ENABLE_PROPERTY_NAME))
         .to(isVerifyEpochHashEnabled);
+    bindConstant().annotatedWith(Names.named(CURRENT_NETWORK)).to(currentNetwork);
```

Comment:
it would be better to bind this in `RadixNodeModule` (alongside `bindConstant().annotatedWith(NetworkId.class).to(networkId);`)

Reply:
Usually, I prefer a self-contained approach. Furthermore, I believe it will break the ApiTest, won't it?

Reply:
Done